ffi/capi/Cargo.toml

```diff
@@ -127,7 +127,7 @@ diplomat-runtime = { workspace = true }
 
 # Optional ICU4X components and their dependent utils
 fixed_decimal = { workspace = true, features = ["ryu"] , optional = true }
-icu_calendar = { workspace = true, features = ["alloc", "ixdtf"], optional = true }
+icu_calendar = { workspace = true, features = ["alloc", "ixdtf", "unstable"], optional = true }
```

Comment:
issue: unstable features should not be auto enabled by other ICU4X crates

There's some discussion about this here: https://github.com/unicode-org/icu4x/issues/6659#issuecomment-3409034218, but the general idea is that unstable things that are needed cross-crate should be `doc(hidden)` (usually, exported through a _different_ path so that people don't accidentally use it without turning the feature on).